From c410b7ca9b74e82c2d053dea4e95151c24d88226 Mon Sep 17 00:00:00 2001
From: Xdoxgg <m.zhuk007@gmail.com>
Date: Wed, 19 Mar 2025 21:07:30 +0300
Subject: [PATCH] Introduce bug in multiply function

---
 app.by | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/app.by b/app.by
index 7cb3a77..66c7078 100644
--- a/app.by
+++ b/app.by
@@ -5,6 +5,7 @@ def add(a, b):
 def subtract(a, b):
     return a - b
 
+
 # app.py
 def multiply(a, b):
-    return a * b
+    return a + b  # Ошибка: должно быть a * b
-- 
2.25.1

